readthedocs/templates/sphinx/layout.html

```diff
@@ -37,7 +37,11 @@
     builder: "sphinx",
     theme: "{{ html_theme }}",
     docroot: "{{ conf_py_path }}",
+    {% if page_source_suffix %} 
+    source_suffix: "{{ page_source_suffix }}",
+    {% else %}
     source_suffix: "{{ source_suffix }}",
+    {% endif %}
```

Comment:
Just a tip: `{% firstof page_source_suffix source_suffix %}` is a viable alternative.